README.md

```diff
@@ -83,40 +83,77 @@ The easiest way to compile the code and run the system locally is using [Docker]
 
 Don't forget to run the docker daemon!
 
+## Build the containers
+
+Building with Docker utilizes multi-stage builds. In order to run an architecture you will need to build each architecture independently if building locally.
+
+**Note:** We have pre-built images available [here](#launch-the-system-with-a-pre-built-image) if you would rather pull the images from GitHub packages over building them locally.
+
+1. Build 2PC architecture:
```

Comment:
Love the separate instructions, but it may be a bit clearer if we have separate sections focused on each architecture instead of separate instructions for each architecture at every step.

Reply:
I don't have strong opinions on this. I think it's reasonable as-is, but @jallen-frb, if you think it will be cleaner as @kylecrawshaw suggests, I think it would be fine to change it as well.